bin/install.sh

```diff
@@ -51,6 +51,21 @@ function install_control() {
     step "auto-injector.."
     bin/iop ${ISTIO_CONTROL_NS} istio-autoinject $IBASE/istio-control/istio-autoinject --set global.istioNamespace=${ISTIO_CONTROL_NS} --set global.istioNamespace=${ISTIO_CONTROL_NS} $RESOURCES_FLAGS
 
+    # Assure that webhook is deleted whenever sidecar-injector is deleted
```

Comment:
Take a look at the makefile, an alternative is to label the namespace as 'do not inject'.
We probably still want to have a 'delete' function that will cleanup the webhook - maybe part of istioctl
or operator.

Reply:
We could simply remove the label, right? But the webhook is outdated anyway once the injector was deleted. So the operator sounds like a good place, imho it should be automatically removed.